apps/backend/.env

```diff
@@ -83,3 +83,5 @@ STACK_OPENAI_API_KEY=# enter your openai api key
 STACK_FEATUREBASE_API_KEY=# enter your featurebase api key
 STACK_STRIPE_SECRET_KEY=# enter your stripe api key
 STACK_STRIPE_WEBHOOK_SECRET=# enter your stripe webhook secret
+STACK_INIT_TELEGRAM_BOT_TOKEN= # enter you telegram bot token
+STACK_INIT_TELEGRAM_CHAT_ID=# enter your telegram chat id
```

Comment:
what does the INIT stand for? why not just STACK_TELEGRAM_BOT_TOKEN?

Reply:
ahh because of init script? I think let's keep it without the INIT, we can add more TG bot actions later in that case